extensions/spatial.sigmf-ext.md

```diff
@@ -0,0 +1,372 @@
+to directional antennas and multichannel phase-coherent datasets used for signal
+direction of arrival and beamforming.
+
+Multichannel datasets can be stored in SigMF Collections or as multichannel
+interleaved data depending on the application. Collections are RECOMMENDED due
+to the easier processing and better application support.
+
+The `spatial` extension makes use of cartesian coordinates to define array
+geometry, and spherical coordinates for reporting bearings. This, coupled with
+the various methods for defining boresights and bearing references can become
+complicated rapidly so the union of these two cordinate reference systems (CRS)
+is illustrated in Figure 1 below. The ISO 80000-2:2019 compliant cartesian and
+spherical systems have a specific relationship with the reported azimuth and
+elevation, which use the conventional geospatial definitions (degrees east of
+north, and degrees above horizon respectively).
+
+![SigMF Spatial Coordinate Reference System](spatial_crs.png)
+
+**Figure 1 - SigMF Spatial Extension Coordinate Reference Systems**
+
+As shown in Figure 1, the boresight of the aperture is defined as being in the
+direction of the positive X-axis. Based on this, as an example, a horizontal
+uniform linear array aperture would be defined along the Y-axis (see examples
+section below).
+
+## 0 Datatypes
+
+This extension defines the following datatypes:
+
+|type|long-form name|description|
+|----|--------------|-----------|
+|bearing|signal direction bearing|Quantitative representation of a direction with optional error.|
+|cartesian_point|cartesian position|Cartesian coordinate triplet [x, y, z]|
+
+### 0.1 The `bearing` Object
+
+The `bearing` field is used to describe relative one or two dimensional signal
+directions. The units are always in degrees; while it is legal for this field to
+have any value it is RECOMMENDED that it be between 0 and 360 or +/- 180.
+
+|name|required|type|units|description|
+|----|--------|----|-----|-----------|
+|`azimuth`|false|float|degrees|Azimuth component of the direction in degrees increasing clockwise.|
+|`elevation`|false|float|degrees|Elevation component of the direction in degrees above horizon.|
+|`range`|false|float|meters|Line-of-sight slant range to emitter, if known, in meters.|
+|`az_error`|false|float|degrees|Error or uncertainty in the azimuth component.|
+|`el_error`|false|float|degrees|Error or uncertainty in the elevation component.|
+|`range_error`|false|float|meters|Error or uncertainty in the range component.|
+
+The `az_error`, `el_error`, and `range_error` field units are degrees, but the
+exact meaning of 'error' in this context is not explicitly defined. Applications
+SHOULD specify their specific meaning, and in general this should be interpreted
+as an uncertainty range. The error fields should be SHOULD NOT be used if the
+corresponding field estimates are not included.
+
+```json
```

Comment:
Since you've already got an image, just an idea, how hard would it be to overlay what this specific example looks like on top of that image? Not a hard req, but, may be helpful to the reader.

Reply:
Going to put this on the 'todo'. representing error/uncertainty can be tricky, especially in a fixed image.

Reply:
Fair 👍🏼